graph/tag.go

```diff
@@ -1,18 +1,6 @@
 package graph
 
-import (
-	"fmt"
+func (s *TagStore) Tag(name string, repo string, tag string, force bool) error {
```

Comment:
Is it worth even keeping this file around any more? The only value I see here is the name Tag() instead of Set() - which better aligns with what an end user sees.  Perhaps we could remove this file and rename tags.go's Set() to Tag() - since its just a wrapper for SetLoad().
@LK4D4 what do you think?